### PR TITLE
Docs: Converting cookbooks and benchmarks chapters

### PR DESCRIPTION
**Pull request by cmills1095**
In doc/sphinx/cookbooks/using-particles: There's a lot of text (several paragraphs, a couple of prm files) in this section of the current iteration of the pdf manual that seems to be missing from the corresponding section of the current iteration of the latex manual. Should I go back and input those sections or was it removed intentionally?

### CODE REVIEW

**Comment by cmills1095** [comment]
Also, to be clear: This is NOT the actual cookbooks or benchmarks, just the text bits at the beginnings of chapters/certain sections which are not associated with a specific cookbook/benchmark

File: doc/sphinx/user/cookbooks/index.md [hunk 1]
```diff
@@ -2,36 +2,35 @@
 # Cookbooks
 In this section, let us present a number of &ldquo;cookbooks&rdquo; &ndash;
 examples of how to use <span class="smallcaps">ASPECT</span> in typical or
-less typical ways. As discussed in Sections&nbsp;[\[sec:running\]][1] and
-[\[sec:parameters\]][2], <span class="smallcaps">ASPECT</span> is driven by
+less typical ways. As discussed in {ref}`cha:run-aspect`, <span class="smallcaps">ASPECT</span> is driven by
```
Comment (gassmoeller) [inline doc/sphinx/user/cookbooks/index.md:5]:
```suggestion
less typical ways. As discussed in {ref}`cha:run-aspect`, ASPECT is driven by
```

File: doc/sphinx/user/cookbooks/teaching-setups.md [hunk 2]
```diff
@@ -10,90 +11,21 @@ more suitable for running them in the classroom, or they include additional
...
-Davies, et al. 2015. &ldquo;A Community Benchmark for Viscoplastic Thermal
-Convection in a 2-d Square Box.&rdquo; *Geochem.&nbsp;Geophys.&nbsp;Geosyst.*
-16: 2175&ndash;96.
-
-</div>
-
-</div>
+1. {ref}`sec:cookbooks:convection-box-particles` (using the files in `cookbooks/convection-box-particles/`)
 
-[1] You can also extend <span class="smallcaps">ASPECT</span> using plugins
```
Comment (gassmoeller) [inline doc/sphinx/user/cookbooks/teaching-setups.md:70]: Where did this note and the notes below go?

**Review by gassmoeller — COMMENTED** [review]
Looks good except for my 2 comments

**Comment by gassmoeller** [comment]
also you will have to fix the indentation

File: doc/sphinx/user/cookbooks/teaching-setups.md (hunk 2 above)
Comment (cmills1095) [reply doc/sphinx/user/cookbooks/teaching-setups.md:70]: The first two went into cookbooks/overview, the third went into benchmarks/index

**Comment by gassmoeller** [comment]
Ok, but indentation is still failing.

**Review by gassmoeller — APPROVED** [review]
Indentation fixed :+1:

**Pull request merged**